src/rules/comment.ts

```diff
@@ -0,0 +1,124 @@
+import { createRule } from '../util';
+
+export const ruleName = 'comment-syntax';
+
+type Options = [];
+
+const SPACE_CHARCODE = 32;
+const NEWLINE_CHARCODE = 10;
+const STAR_CHARCODE = 42;
+
+type MessageIds =
+  | 'lineCommentCapital'
+  | 'lineCommentEnding'
+  | 'blockCommentCapital'
+  | 'blockCommentEnding'
+  | 'shouldStartWithSpace'
+  | 'shouldStartWithBlock';
+
+export default createRule<Options, MessageIds>({
+  name: ruleName,
+  meta: {
+    type: 'layout',
+    docs: {
+      category: 'Best Practices',
+      description: `Enforce block comments to have capital first letter and end with dots and 
+      line comments no capital first letter and no dot`,
+      recommended: 'error',
+    },
+    fixable: 'code',
+    schema: [],
+    messages: {
+      shouldStartWithSpace: 'A line comment should start with a space',
+      shouldStartWithBlock: 'A block comment should start with /**\n *',
+      lineCommentCapital: 'A line comment cannot start with a capital letter unless the entire word is',
```

Comment:
Did you mean to add more text here? It just ends with `...unless the entire word is`

Reply:
Not really. `It cannot start with a capital letter unless the entire word is capital` better?